extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -153,7 +153,7 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
             }
             return init;
         } else if (xhr) { // XHR
-            if (this.currentWindowHost && CorrelationIdHelper.canIncludeCorrelationHeader(this._config, xhr.ajaxData.getAbsoluteUrl(),
+            if (CorrelationIdHelper.canIncludeCorrelationHeader(this._config, xhr.ajaxData.getAbsoluteUrl(),
```

Comment:
If `this.currentWindowHost` is empty, then `canIncludeCorrelationHeader` will return false with this check [here ](https://github.com/microsoft/ApplicationInsights-JS/blob/e86462c8a430717d87f01c1864de890dfe29a979/shared/AppInsightsCommon/src/Util.ts#L646) then this if check is still false correct? Unless `requestHost` is empty too?

Reply:
Yes I don't think it will affect that check at all unless `requestHost` is somehow falsy as well. Since it comes from an actual request being made I don't think its possible. From what I can tell, the `this.currentWindow host &&` check isn't actually useful.

Reply:
yeah I agree to remove it, I mean if the case happens again (`this.currentWindowHost` is empty), it still can't pass this if check, the logic under if will not run as before. Is that the expected behavior?

Reply:
I've modified it to not pass the if check anymore unless enableCorsCorrelation is enable and it also passes the included/excluded domains check. I consider an empty `currentWindowHost` an undefined scenario, so my thought it to always reject it unless the user has opted into a particular CORS correlation scenario.